### ISSUE TITLE
Configure which schemas models are materialized in

### ISSUE THREAD

**Issue by drewbanin** [fishtown-analytics/dbt#284]
Semi-related: https://github.com/analyst-collective/dbt/issues/198

This would be useful for both organization and permission configuration

**Comment by clausherther** [comment]
+1!

**Comment by atharvai** [comment]
This would be great to have in addition to default target as right now in `profiles.yml` .

I imagine this to be something like a configuration value for model.

```python
{{
  config(
    materialized = 'table',
    schema = 'dbt'
  )
}}```

`schema` keyword is also used for `schema.yml` but should be clear enough from context what is being referenced.

**Comment by adamhaney** [comment]
Would it be possible to pass the schema as either a string or a dictionary that was keyed on target name (ie, dev/prod) this would potentially allow for schema targets to be configured without breaking multiple environments?

**Comment by igrayson** [comment]
This directly approaches my use case @ https://getdbt.slack.com/archives/C0VLZM3U2/p1497459181879656

**Comment by drewbanin** [comment]
Happy to report that this has been [merged](https://github.com/fishtown-analytics/dbt/pull/522) into `development` and will be available in dbt 0.9.0.

We'll be publishing our second 0.9.0 alpha release (including this feature) in the next day or two -- please let me know if you're interesting in trying it out!

Relevant docs: https://dbt.readme.io/v0.9/docs/using-custom-schemas
